packages/web/config/ibc-assets.ts

```diff
@@ -1999,6 +1999,14 @@ export const IBCAssetInfos: (IBCAsset & {
         coinMinimalDenom: "uqsomm",
         isVerified: true,
       },
+      {
+        counterpartyChainId: "cosmoshub-4",
+        sourceChannelId: "channel-4",
+        destChannelId: "channel-141",
```

Comment:
this doesn't look right to me. Please register the IBC connection to the Chain Registry.

Reply:
Hi @JeremyParish69 , FX token will cross the chain to osmosis via cosmoshub, so it is registered as a channel and ibc token on cosmoshub